src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .trim()
+    .split(';')
+    .filter(Boolean)
+    .reduce((acc, rule) => {
+      const [property, value] = rule.split(':').map((part) => part.trim());
+
+      return { ...acc, [property]: value };
```

Comment:
[CODE STYLE] - The chained methods starting from line 9 should have the same indentation to improve the readability of the code. Align the methods vertically.